public/javascripts/data.js

```diff
@@ -433,7 +441,66 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
             return;
         }
 
-        // metadata is special
+         // control type "Audit"
+         if (control.type == 'audit') {
+
+            /* Add the `orx:audit` element to the `orx:meta` element.
+             * 
+             * TODO: explicitly select the element by its name 'orx:meta'
+             * 
+             * Discuss: why are other supported meta elements not added to `orx:meta`?
+             * https://getodk.github.io/xforms-spec/#metadata
+             * 
+             * The first three attributes are required for location audit.
+             * location_priority == "off" disables location audit and excludes all three location audit fields.
+             */
+
+            if (instance.children[0].name == 'orx:meta') {
+                instance.children[0].children.push({ name: 'orx:audit' });
+            } else {
+                console.log("No 'orx:meta' element found in instance. Adding audit will likely fail.");
+            }
+
+            /* Add binding node with parameters 
+             * 
+             * Special path: /data/meta/audit
+             * Conditional logic to drop location audit fields if location_priority == "off"
+             */
+            var binding = {
+                name: 'bind',
+                attrs: {
+                    'nodeset': xpath + 'meta/' + control.name,
+                    type: 'binary'
+                }
+            };
+
+            if (control.location_priority != undefined && control.location_priority != 'off') {
+                /* Enable location audit with sane defaults if missing */
+                binding.attrs['odk:location-priority'] = control.location_priority;
+                if (control.location_min_interval != undefined) {
+                    binding.attrs['odk:location-min-interval'] = control.location_min_interval;
+                } else {
+                    binding.attrs['odk:location-min-interval'] = '60';
+                };
+                if (control.location_max_age != undefined) {
+                    binding.attrs['odk:location-max-age'] = control.location_max_age;
+                } else {
+                    binding.attrs['odk:location-max-age'] = '600';
+                };
+            } else {
+                console.log("Location audit disabled");
+            };
+
+            if (control.track_changes != undefined) {
+                binding.attrs['odk:track-changes'] = control.track_changes;
+            };
+
+            model.children.push(binding);
+
+            return;
+        }
```

Comment:
This pushes two elements into the exported XML, meta and bind node. See inline comments.